### ISSUE TITLE
Highest Points (Pro) is throwing errors in test_highest_points_pro

### ISSUE THREAD

**Issue by mfunk** [Esri/military-tools-geoprocessing-toolbox#68]
This is a problem in with the model in the Pro toolbox **Military_Tools.tbx**, and is not an error in Desktop.

```
Testing Highest Points (Pro).
Running tool (Highest Points)
ERROR: Start Time: Thursday, May 19, 2016 7:36:10 AM
Failed to execute. Parameters are not valid.
ERROR 000840: The value is not a Raster Layer.
ERROR 000735: Input Area: Value is required
The value is empty.
ERROR 000735: Input Features: Value is required
ERROR 000860: Input raster: is not the type of Composite Geodataset, or does not exist.
ERROR 000860: Input raster: is not the type of Composite Geodataset, or does not exist.
Failed to execute (HighestPoints).
Failed at Thursday, May 19, 2016 7:36:11 AM (Elapsed Time: 0.44 seconds)
Start Time: Thursday, May 19, 2016 7:36:10 AM
Failed to execute. Parameters are not valid.
ERROR 000840: The value is not a Raster Layer.
ERROR 000735: Input Area: Value is required
The value is empty.
ERROR 000735: Input Features: Value is required
ERROR 000860: Input raster: is not the type of Composite Geodataset, or does not exist.
ERROR 000860: Input raster: is not the type of Composite Geodataset, or does not exist.
Failed to execute (HighestPoints).
Failed at Thursday, May 19, 2016 7:36:11 AM (Elapsed Time: 0.44 seconds)
```

THe issue where the tests do not fail is a separate one: #66.

**Comment by mfunk** [comment]
OK this is failing because of a hardcoded reference to *MyProject13* in variable "Highest Point Feature Class". The value of the variable should be empty, allowing the user to type an output name (and the path default to their system), or give a default name "HighestPoints" without a path/scratch/etc (which means the system will default the path as well).

**Comment by mfunk** [comment]
All so found a miss named variable in Raster Calculator expression:

```
Con("%intermed2%" == %MAXIMUM%, "%intermed2%")

```

it should be:

```
Con("%Temp Raster%" == %MAXIMUM%, "%Temp Raster%")
```

After this the tool runs in Pro:

```
...
Testing Highest Points (Pro).
Running tool (Highest Points)
ArcPy works
Spatial checked out
Testing Lowest Points (Pro).
...

```

**Comment by mfunk** [comment]
Fixed in #70 

```
Python 3.4 Tests ===============================
Debug messaging is OFF
Logging results to: MTGT_2016-May-19_15-12-34_seq0.log
adding test: test_convert_coordinates_pro
adding test: test_table_to_twopointline_pro
adding test: test_table_to_ellipse_pro
adding test: test_table_to_lineofbearing_pro
adding test: test_table_to_point_pro
adding test: test_table_to_polygon_pro
adding test: test_table_to_polyline_pro
adding test: test_find_local_peaks_pro
adding test: test_highest_points_pro
adding test: test_lowest_points_pro
running 10 tests...
ArcPy works
Testing Convert Coordinates (Pro).
ArcPy works
Testing Table To 2-Point Line (Pro).
ArcPy works
Testing Farthest On Circle (Pro).
ArcPy works
Testing Table To Line Of Bearing (Pro).
ArcPy works
Testing Table To Point (Pro).
Running tool (Table To Point)
ArcPy works
Testing Table To Polygon (Pro).
ArcPy works
Testing Table To Polyline (Pro).
ArcPy works
Spatial checked out
Testing Find Local Peaks (Pro).
Running tool (Find Local Peaks)
ArcPy works
Spatial checked out
Testing Highest Points (Pro).
Running tool (Highest Points)
ArcPy works
Spatial checked out
Testing Lowest Points (Pro).
Running tool (Lowest Points)
6
Test success: True
RESULTS =================================================

Number of tests run: 10
Number of errors: 0
Number of failures: 0

END OF TEST =========================================
```
